node/agent_proxy.go

```diff
@@ -218,6 +138,7 @@ func (a *HostAgent) ReportHealthStatus(req dao.HealthStatusRequest, unused *int)
 
 // ReportInstanceDead proxies ReportInstanceDead to the master server.
 func (a *HostAgent) ReportInstanceDead(req dao.ServiceInstanceRequest, unused *int) error {
+	// FIXME: use new master.ClientInterface instead
```

Comment:
Yes - I'm in the process of moving all of `dao/elasticsearch/health.go` stuff to `rpc`